ansible/roles/openshift_firstboot_scripts/files/disable-hyperthreading.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo off > /sys/devices/system/cpu/smt/control
```

Comment:
I think we should add the grubby command here in case the machine reboots.

Reply:
I added a grubby command to add this.